dev-lang/hy/hy-0.19.0.ebuild

```
@@ -1,10 +1,10 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
 
 RESTRICT="test" # needs some pointy sticks. Seriously.
-PYTHON_COMPAT=(python3_7)
+PYTHON_COMPAT=(python{3_7,3_8})
```

Can you add Python 3.9 too?

Also, style:
```suggestion
PYTHON_COMPAT=( python3_{7,8} )
```

---

Python3_9 is blocked by funcparserlib, which works with 3.9 itself, but has no 3_9 in ebuild's PYTHON_COMPAT. Waiting https://github.com/vlasovskikh/funcparserlib/pull/63 to be merged.

---

If it works (and tests pass with FEATURES=test), feel free to add it in this PR! We're _trying_ to be more ahead of the curve this time around. I think we're getting there.

If tests pass for you, that's good enough for us. I'll check too when merging.

---

Ah, opened funcparserlib's ebuild and it is already bumped :smile:. Ok, now we can just add 3.9 and test.

---

I've also added hy-0.20.0 ebuild and it's working too, but have same issues with tests.